package.json

```diff
@@ -20,13 +20,14 @@
     "dotenv": "^16.3.1",
     "eslint-config-next": "13.5.2",
     "eslint-import-resolver-typescript": "^3.6.1",
-    "next": "13.5.2",
+    "next": "13.5.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.8"
+    "styled-components": "^6.0.9"
   },
   "devDependencies": {
     "@calblueprint/eslint-config-react": "^0.0.3",
+    "@types/styled-components": "^5.1.28",
```

Comment:
Do you remember what this was for?

Reply:
My import of styled components was erroring, and it told me to add this line! however, the red error line still appears, even with this line included :<

Reply:
Weird... let's look into it tomorrow